go.mod

```diff
@@ -112,6 +112,7 @@ require (
 	github.com/kr/pretty v0.3.1
 	github.com/kr/text v0.2.0
 	github.com/nsf/jsondiff v0.0.0-20210926074059-1e845ec5d249
+	github.com/shirou/gopsutil v3.21.11+incompatible
```

Comment:
Any reason for not using the latest version here? Looks like this one is from late 2021 and there's a bunch more recent ones.

Reply:
The import likely needs to be changed to the following for this (and to also fix the other comment):

```
github.com/shirou/gopsutil/v3 v3.23.1
```

Reply:
@dbussink no good reason, this is what `go mod tidy` chose maybe due to not supporting version subdirs

Updating to the version suggested 👍